src/main/java/com/mraof/minestuck/blockentity/IntellibeamLaserstationBlockEntity.java

```diff
@@ -0,0 +1,196 @@
+package com.mraof.minestuck.blockentity;
+
+import com.mraof.minestuck.alchemy.AlchemyHelper;
+import com.mraof.minestuck.block.MSBlocks;
+import com.mraof.minestuck.block.machine.IntellibeamLaserstationBlock;
+import com.mraof.minestuck.item.MSItems;
+import com.mraof.minestuck.util.MSSoundEvents;
+import net.minecraft.ChatFormatting;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.protocol.Packet;
+import net.minecraft.network.protocol.game.ClientGamePacketListener;
+import net.minecraft.network.protocol.game.ClientboundBlockEntityDataPacket;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.Containers;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class IntellibeamLaserstationBlockEntity extends BlockEntity
+{
+	protected ItemStack card = ItemStack.EMPTY;
+	protected int EXP_LEVEL_CAPACITY = 10;
+	protected int EXPERIENCE_LEVEL = 0;
+	protected float SOUND_SCALER = 0F;
+	protected static int WAIT_TIMER = 0;
```

Comment:
Two things about this section:
1. SCREAMING_SNAKE_CASE is usually only appropriate for static final variables, which remain unchanged while the code is running. The naming scheme for EXP_LEVEL_CAPACITY is appropriate due to this definition, and as a result should be changed from protected to private static final or public static final
2. The use of the protected access modifier means that these values can be edited by other Classes in this package. While I dont know if their usage was intentional, private would be most appropriate considering the nature of them and the fact that they are only currently used in this class

Heres a useful explanation of how the common modifiers work: https://stackoverflow.com/questions/5083232/usage-patterns-for-private-static-final-public-abstract-keywords-in-java

Reply:
done